cmd/rac/cli.go

```diff
@@ -95,8 +109,10 @@ func processFiles(
 func init() {
 	skipImages = flag.Bool("skip-images", false, "Extract images from rac-files.\n(Default: false)")
```

Comment:
You are often using the bools with double negation "Don't skip images" - perhaps change the bool to be positive "images" true or false. This mean the user says -skip-images but we are using the negation.